packages/logging/src/handlers/log.handler.ts

```diff
@@ -7,37 +7,86 @@ import {ServiceDefinitionTagEnum, tag, TracingContext, InternalContainerParamete
 import {LogHandlerInterface} from "../interfaces/log-handler.interface";
 import {Utils} from "../utils/utils";
 
+/**
+ * The LogHandler to use when we want to output some logs.
+ * This handler makes sure that only the right level of logs are outputted, stacks logs, and logs with different loggers.
+ * It is registered with the tag LogHandlerInterface so that it can be injected as a LogHandlerInterface to facilitate mocking.
+ */
 @tag("LogHandlerInterface")
```

Comment:
Manque le moduleScoped.

Reply:
Me souviens pu pourquoi on les mettait pi des foi snon, messemble que c'est dans la doc haha

Reply:
We usually put it on the ones that have a tag from ServiceDefinitionTagEnum since we inject an array and don't want it to be injected if it's not imported. Here it's not 100% necessary because we would only inject it using `LogHandlerInterface`